Makefile

```diff
@@ -124,8 +124,20 @@ test-python:
 
 test-python-ci:
 	make build-platform-assets
-	@echo "--> Running CI Python tests"
-	pytest tests/integration tests/sentry --cov . --cov-report="xml:.artifacts/python.coverage.xml" --junit-xml=".artifacts/python.junit.xml" || exit 1
+	@echo "--> Running backend tests, except those that require Kafka"
+	pytest tests/integration tests/sentry \
+		--ignore tests/sentry/eventstream/kafka \
```

Comment:
hmm `tests/sentry/eventstream/kafka/test_consumer.py` is also run by `snuba test` i'm just learning, where kafka is already run, yet it fails similarly

Reply:
also still some backend tests requiring kakfa that i missed:

```
tests/sentry/incidents/endpoints/test_organization_alert_rule_index.py
tests/sentry/incidents/endpoints/test_project_alert_rule_index.py
tests/sentry/incidents/test_subscription_processor.py
```

Reply:
jk so for example tests/sentry/incidents/test_subscription_processor.py passes w/o kafka, wtf `https://github.com/getsentry/sentry/runs/6532407273?check_suite_focus=true`

Reply:
Okay all these tests pass locally without kafka. It's definitely my changes to kafka stuff htat's the issue.

Reply:
Okay so those tests are still failing in CI, I fixed some kafka ones though except `tests/sentry/ingest/ingest_consumer/test_ingest_consumer_kafka.py::test_ingest_topic_can_be_overridden`